examples/ui.content/src/main/content/jcr_root/content/core-components-examples/library/adaptive-form/pagefooter/.content.xml

```diff
@@ -0,0 +1,145 @@
+          textIsRich="true"/>
+        <teaser
+          cq:styleIds="[1550165685463]"
+          jcr:created="{Date}2019-02-14T16:41:54.952+01:00"
+          jcr:createdBy="admin"
+          jcr:description="&lt;p>GitHub&lt;/p>&#xd;&#xa;"
+          jcr:lastModified="{Date}2019-02-14T16:49:57.290+01:00"
+          jcr:lastModifiedBy="admin"
+          jcr:primaryType="nt:unstructured"
+          jcr:title="Technical Documentation"
+          sling:resourceType="core-components-examples/components/teaser"
+          actionsEnabled="false"
+          descriptionFromPage="false"
+          linkURL="https://github.com/adobe/aem-core-forms-components/tree/master/ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/pagefooter/v1/pagefooter"
+          textIsRich="true"
+          titleFromPage="false"/>
+        <title_865328580
+          cq:styleIds="[1544759664489]"
+          jcr:created="{Date}2018-12-06T19:22:23.729+01:00"
+          jcr:createdBy="admin"
+          jcr:lastModified="{Date}2018-12-06T19:23:25.116+01:00"
+          jcr:lastModifiedBy="admin"
+          jcr:primaryType="nt:unstructured"
+          jcr:title="Examples"
+          sling:resourceType="core/wcm/components/title/v2/title"
+          type="h2"/>
+        <title_290384408
+          cq:styleIds="[1544759676459]"
+          jcr:created="{Date}2018-12-06T19:22:48.620+01:00"
+          jcr:createdBy="admin"
+          jcr:lastModified="{Date}2019-05-22T11:01:40.926+03:00"
+          jcr:lastModifiedBy="admin"
+          jcr:primaryType="nt:unstructured"
+          jcr:title="Standard"
+          sling:resourceType="core/wcm/components/title/v2/title"
+          type="h3"/>
+        <text_1337506761
+          jcr:created="{Date}2018-12-06T19:23:41.968+01:00"
+          jcr:createdBy="admin"
+          jcr:lastModified="{Date}2019-05-22T11:26:32.604+03:00"
+          jcr:lastModifiedBy="admin"
+          jcr:primaryType="nt:unstructured"
+          sling:resourceType="core/wcm/components/text/v2/text"
+          text="&lt;p>The component, simply displaying placeholder data.&lt;/p>"
+          textIsRich="true"/>
+        <demo
+          jcr:created="{Date}2018-12-07T12:55:03.496+01:00"
+          jcr:createdBy="admin"
+          jcr:lastModified="{Date}2018-12-07T12:55:03.496+01:00"
+          jcr:lastModifiedBy="admin"
+          jcr:primaryType="nt:unstructured"
+          sling:resourceType="forms-components-examples/components/demo">
+          <component
+            jcr:primaryType="nt:unstructured"
+            sling:resourceType="forms-components-examples/components/demo/component">
+            <guideContainer
+              fd:version="2.1"
+              fieldType="form"
+              jcr:primaryType="nt:unstructured"
+              sling:resourceType="forms-components-examples/components/form/container">
+              <container
+                jcr:primaryType="nt:unstructured"
+                fieldType="demo-component"
```

Comment:
why fieldType is necessary?

Reply:
Field Type is a required field in Forms core components, or else it breaks.

Reply:
This is not required @barshat7. It should not break